### DISCUSSION TITLE
How to support platform X?

### DISCUSSION THREAD

**Discussion by z0al** [react-native-community/discussions-and-proposals#487] [Ideas]
With the new architecture being rolled out should we expect the [Out-of-Tree Platforms](https://reactnative.dev/docs/out-of-tree-platforms) doc to get the love it deserves?

I was entertaining the idea of experimenting with RN Linux support based GTK4 for the sake of learning but honestly had to give up as I couldn't find any resources. I've read all about "custom React renderer" docs/tutorials but these are not very useful when it comes to RN.

Any guidance or links to resources would be very much appreciated.

**Comment by cortinico** [comment]
> With the new architecture being rolled out should we expect the [Out-of-Tree Platforms](https://reactnative.dev/docs/out-of-tree-platforms) doc to get the love it deserves?

I agree that there is space for improvements on this side. Specifically, we could start by adding the various variations of React Native Skia + https://github.com/react-native-tvos/react-native-tvos and similar projects.

This would make those projects easier to discover in general.

If you're looking for specific documentation on how to onboard a new platform, that's something that would be totally valuable but we're not actively working on it at the moment. Any community effort in this sense though would be more than appreciated as a starting point.

**Reply by Kudo** [reply]
that is the skia project based on fabric https://github.com/react-native-skia/react-native-skia to support new platforms. originally, it's just my POC project and many functionalities are not yet implemented. nagra's fork is more mature, especially for linux support: https://github.com/nagra-opentv/react-native-skia. you can check their videos in the tweets: https://twitter.com/munezbn/status/1445085416089001985

**Reply by cortinico** [reply]
@Kudo would you be up for adding them to the community-supported platforms on the website?
